src/services/apps-metadata.ts

```diff
@@ -8,6 +8,9 @@
 import { getErrorMessage } from '../utils/error';
 import { Url } from '../utils/api-rest';
 
+//import { IdpSettings } from '@gridsuite/commons-ui';
+export type IdpSettingsJson = typeof import('../../public/idpSettings.json');
+
 export type EnvJson = typeof import('../../public/env.json') & {
```

Comment:
Env is application specific that's why we can import it from env.json file, but for IdpSettings it is regulated by commons-ui and the way it should be handled in authenticiation library so I prefer we remain based on the type defined in commons-ui